tests/baselines/reference/privateNameDuplicateField.errors.txt

```diff
@@ -1,114 +1,657 @@
-tests/cases/conformance/classes/members/privateNames/privateNameDuplicateField.ts(3,5): error TS2300: Duplicate identifier '#foo'.
-tests/cases/conformance/classes/members/privateNames/privateNameDuplicateField.ts(7,5): error TS2300: Duplicate identifier '#foo'.
-tests/cases/conformance/classes/members/privateNames/privateNameDuplicateField.ts(8,5): error TS2300: Duplicate identifier '#foo'.
-tests/cases/conformance/classes/members/privateNames/privateNameDuplicateField.ts(13,5): error TS2300: Duplicate identifier '#foo'.
-tests/cases/conformance/classes/members/privateNames/privateNameDuplicateField.ts(14,9): error TS2300: Duplicate identifier '#foo'.
-tests/cases/conformance/classes/members/privateNames/privateNameDuplicateField.ts(18,5): error TS2393: Duplicate function implementation.
-tests/cases/conformance/classes/members/privateNames/privateNameDuplicateField.ts(19,5): error TS2393: Duplicate function implementation.
-tests/cases/conformance/classes/members/privateNames/privateNameDuplicateField.ts(24,5): error TS2300: Duplicate identifier '#foo'.
-tests/cases/conformance/classes/members/privateNames/privateNameDuplicateField.ts(25,9): error TS2300: Duplicate identifier '#foo'.
-tests/cases/conformance/classes/members/privateNames/privateNameDuplicateField.ts(30,5): error TS2300: Duplicate identifier '#foo'.
-tests/cases/conformance/classes/members/privateNames/privateNameDuplicateField.ts(31,5): error TS2300: Duplicate identifier '#foo'.
-tests/cases/conformance/classes/members/privateNames/privateNameDuplicateField.ts(32,9): error TS2300: Duplicate identifier '#foo'.
-tests/cases/conformance/classes/members/privateNames/privateNameDuplicateField.ts(36,9): error TS2300: Duplicate identifier '#foo'.
-tests/cases/conformance/classes/members/privateNames/privateNameDuplicateField.ts(37,9): error TS2300: Duplicate identifier '#foo'.
-tests/cases/conformance/classes/members/privateNames/privateNameDuplicateField.ts(41,9): error TS2300: Duplicate identifier '#foo'.
-tests/cases/conformance/classes/members/privateNames/privateNameDuplicateField.ts(42,9): error TS2300: Duplicate identifier '#foo'.
-tests/cases/conformance/classes/members/privateNames/privateNameDuplicateField.ts(46,9): error TS2300: Duplicate identifier '#foo'.
-tests/cases/conformance/classes/members/privateNames/privateNameDuplicateField.ts(47,9): error TS2300: Duplicate identifier '#foo'.
-tests/cases/conformance/classes/members/privateNames/privateNameDuplicateField.ts(48,9): error TS2300: Duplicate identifier '#foo'.
-tests/cases/conformance/classes/members/privateNames/privateNameDuplicateField.ts(49,9): error TS2300: Duplicate identifier '#foo'.
+tests/cases/conformance/classes/members/privateNames/privateNameDuplicateField.ts(6,9): error TS2300: Duplicate identifier '#foo'.

```

Comment:
Haha what happened here? I this because we replaced the early `return` with `continue` in the checker?

Reply:
Nope. Since duplication detection was so unreliable (combinations of type and order matter)  I deceded to add all possible combinations between instance and static private class elements.

Reply:
Nope. Since duplication detection was so unreliable (combinations of type and order matter) I decided to add all possible combinations between instance and static private class elements.